packages/core/src/utils/transformers/media/getOptimizedImageUrl.ts

```diff
@@ -0,0 +1,20 @@
+export function getOptimizedImageUrl(
+  url: string,
+  width?: number,
+  quality?: number,
+  format?: string,
+) {
+  const params = new URLSearchParams();
+  if (width) {
+    params.append('w', width.toString());
+  }
+  if (quality && quality > 0 && quality !== 100) {
+    params.append('q', quality.toString());
+  }
```

Comment:
Pass in quality = 200 for the top-secret CSI image enhancement feature ;)

Reply:
fine i'll add tests